ios/.swiftlint.yml

```diff
@@ -7,6 +7,7 @@ excluded:
   - KMMICTests/Sources/Mocks/Sourcery/AutoMockable.generated.swift
   - KMMIC/Sources/Supports/Helpers/Rswift/R.generated.swift
   - KMMIC/Sources/Supports/Helpers/Shared/Shared.generated.swift
+  - KMMIC/Sources/Supports/Helpers/Shared/Shared.tmp.generated.swift
```

Comment:
How can we generate this? 🤔

Reply:
We generated it in a build script
![CleanShot 2023-04-24 at 15 10 09](https://user-images.githubusercontent.com/17875522/233937287-a14eeb68-7ac3-4c1a-9091-608c65dffc0d.png)

Reply:
@suho More information in the `Insight` section
```
After several tries, we cannot combine the configuration of AutoMockable & AutoSharedUsecase, and we end up creating a new .shared-sourcery.yml then after triggering a build in Xcode

- Run a build script to generate the AutoSharedUseCase.generated.swift
- Run the normal Sourcery build script to generate the AutoMockable.generated.swift
```